packages/ui5-task-zipper/readme.md

```diff
@@ -0,0 +1,60 @@
+# UI5 task for transpiling ES6+ sources
+
+Task for [ui5-builder](https://github.com/SAP/ui5-builder), enabling transpiling.
+
+## Install
+
+```bash
+npm install ui5-task-zipper --save-dev
+```
+
+## Configuration options (in `$yourapp/ui5.yaml`)
+
+- debug: true|false  
+Verbose logging
+
+- archiveName: `String`  
+Desired name for the `.zip` archive. 
+Default value: `<app-id.zip>`
+
+## Usage
+
+1. Define the dependency in `$yourapp/package.json`:
+
+```json
+"devDependencies": {
+    // ...
+    "ui5-task-zipper": "*"
+    // ...
+},
+"ui5": {
+  "dependencies": [
+    // ...
+    "ui5-task-zipper",
+    // ...
+  ]
+}
+```
+
+> As the devDependencies are not recognized by the UI5 tooling, they need to be listed in the `ui5 > dependencies` array. In addition, once using the `ui5 > dependencies` array you need to list all UI5 tooling relevant dependencies.
+
+2. configure it in `$yourapp/ui5.yaml`:
+
+```yaml
+builder:
+  customTasks:
+  - name: ui5-task-zipper
+    afterTask: uglify
+    configuration:
+      archiveName: "webapp"
+```
+
+## How it works
+
+The task can be used to zip all resources in an archive.
+
+## License
+
+This work is [dual-licensed](../../LICENSE) under Apache 2.0 and the Derived Beer-ware License. The official license will be Apache 2.0 but finally you can choose between one of them if you use this work.
+
+When you like this stuff, buy [@vobu](https://twitter.com/vobu) or [@IObert_](https://twitter.com/IObert_) a beer or buy [@pmuessig](https://twitter.com/pmuessig) a coke when you see them.
```

Comment:
You should add your favorite drink here... ;-)

Reply:
Isn't beer a favorite drink 🤔?